R/derive_merged.R

```diff
@@ -853,34 +853,52 @@ get_not_mapped <- function() {
   admiral_environment$nmap
 }
 
-#' Merge a Summary Variable
+#' Merge Summary Variables
 #'
 #' @description Merge a summary variable from a dataset to the input dataset.
 #'
-#' **Note:** This is a wrapper function for the more generic `derive_vars_merged`.
-#'
 #' @param dataset Input dataset
 #'
 #'   The variables specified by the `by_vars` argument are expected.
 #'
 #' @param dataset_add Additional dataset
 #'
-#'   The variables specified by the `by_vars` and the `analysis_var` arguments
-#'   are expected.
+#'   The variables specified by the `by_vars` and the variables used on the left
+#'   hand sides of the `new_vars` arguments are expected.
 #'
 #' @param new_var Variable to add
 #'
+#'  `r lifecycle::badge("deprecated")` Please use `new_vars` instead.
+#'
 #'   The specified variable is added to the input dataset (`dataset`) and set to
 #'   the summarized values.
 #'
 #' @param by_vars Grouping variables
 #'
-#'   The values of `analysis_var` are summarized by the specified variables. The
-#'   summarized values are merged to the input dataset (`dataset`) by the
-#'   specified by variables.
+#'   The expressions on the left hand sides of `new_vars` are evaluated by the
+#'   specified variables. The resulting values are merged to the input dataset
+#'   (`dataset`) by the specified by variables.
```

Comment:
```suggestion
#'   (`dataset`) by the specified **by variables**.
```

I got confused

Reply:
The by variables are used twice. I have emphasized both.

Reply:
I also added more details to "Details".